_data/library/testlibs.yml

```diff
@@ -1,5 +1,9 @@
 - topic: Test frameworks
   libraries:
+    - name: JUnit
+      url: https://junit.org/junit5/
```

Comment:
The URL should point to https://junit.org/junit4/, since we implement the API of JUnit 4, not 5.

Reply:
Addressed feedback.